lix/source-code/client/src/demo.ipynb

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Where do we use this file?

Reply:
@martin-lysk  this is part of  interactive lix documentation, its still mostly empty just added here as a starting point